docs/features/cdc/cdc-querying-streams.rst

```diff
@@ -215,3 +226,95 @@ Note that after upgrading the cluster to 4.4, all new generations (which are cre
 
 .. note::
    We highly recommend using the newest releases of our client CDC libraries (`Java CDC library <https://github.com/scylladb/scylla-cdc-java>`_, `Go CDC library <https://github.com/scylladb/scylla-cdc-go>`_, `Rust CDC library <https://github.com/scylladb/scylla-cdc-rust>`_). They take care of correctly querying the stream description tables and they handle the upgrade procedure for you.
+
+.. _tablets-based-keyspaces:
+
+Tablets-based keyspaces
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To list all available CDC streams for a tablets-based keyspace:
+
```

Comment:
Should start by explaining the schema and columns, not dive in directly to queries.